harness/determined/cli/experiment.py

```diff
@@ -620,7 +676,7 @@ def set_gc_policy(args: Namespace) -> None:
                 render.format_resources(c["resources"]),
             ]
             for c in sorted(checkpoints, key=lambda c: (c["trial_id"], c["end_time"]))
-            if "step" in c and c["step"].get("validation") is not None
+            if "step" in c and c["step"].get("validation")
```

Comment:
I was removing a bunch of `is not None` code elsewhere and this got caught up in it. Mostly I'm not touching this code since it is still using the old trial / step JSON